src/index.css

```
@@ -11,3 +11,23 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+fieldset {
+  border: none;
+}
+
+input{
+  width: 50%;
+  padding: 10px;
+  border-radius: 5px;
+  border: 1px solid #ccc;
+  margin-bottom: 10px;
+  box-sizing: border-box;
+}
+
+button{
```

Please make sure you format all your files before you commit your changes.
```suggestion
button {
```

---

Have we configured his IDE formatting settings yet? @colsen-neslo

---

This should be setup by default.